tutorials/outflux.md

```diff
@@ -0,0 +1,174 @@
+The `influx` client assumes the server is available at http://localhost:8086 by default.
+
+## Schema Discovery, Validation and Transfer
+
+One of Outflux’s features is the ability to discover the schema of an InfluxDB measurement and either validate that a TimescaleDB table exists that can hold the transferred data, or create a new table that can satisfy the schema requirements. 
+
+We can now create a TimescaleDB hypertable ready to receive the demo data we inserted into the InfluxDB instance. If you followed the tutorial and inserted the data from the example, there should be a `taxi` measurement in the `outflux_tutorial` database in the InfluxDB instance. 
+
+The `schema-transfer` command of Outflux can work with 4 schema strategies: 
+* `ValidateOnly`: checks if a specified TimescaleDB database has a hypertable with the proper columns and if it’s partitioned properly, but will not perform modifications
+* `CreateIfMissing`: validates everything if it exists, and it will check if the TimescaleDB  extension is installed (if the hypertable is missing, it will create it  and partition it properly)
+* `DropAndCreate`: drops any existing table with the same name as the measurement, and creates a new hypertable and partitions it properly
+* `DropCascadeAndCreate`: performs the same action as DropAndCreate with the additional strength of executing a cascade table drop if there is an existing table with the same name as the measurement
+
+The `schema-transfer` can be called with parameters that specify the schema strategy, and if the user wants the tags or fields of a measure to be transferred as a single JSONB column. By default each tag and each field is created as a separate column. 
+
+We can run `schema-transfer` with Outflux on our example data with:
+
+```bash
+$ outflux schema-transfer outflux_tutorial taxi \ 
+> --input-server=http://localhost:8086 \ 
+> --output-conn="dbname=postgres user=postgres"
+```
+
+The `schema-transfer` command is executed by specifying the database (`outflux_tutorial`) and then the measurements (`taxi`). If no measurements are specified, all measurements in a database will be transferred. The location of the InfluxDB server is specified with the `--input-server` flag. The target database and other connection options are specified with the `--output-conn` flag. Here we’re using the postgres user and database to connect to our server. How to specify usernames, passwords and many more configuration options about the input and output connections (including which environment variables are recognized) can be discovered on the [GitHub repo for Outflux](https://github.com/timescale/outflux#connection). By default, `schema-transfer` executes with the `CreateIfMissing` strategy. 
```

Comment:
This long line should be broken up (hard to comment on sections otherwise).  "Here we're using the postgres..."  Should "postgres" be in backtick quotes?